packages/core/src/test/testUtil.ts

```diff
@@ -109,6 +109,10 @@ export async function createTestWorkspace(
          * the subDir where the workspace folder will point to within the temp folder
          */
         subDir?: string
+        /**
+         * overridden file name
+         */
+        fileNameOverride?: string
```

Comment:
Notice the other field names. They don't have "override". Please avoid verbosity.
```suggestion
        filename?: string
```